apps/web/src/components/digest-demo-flow/DigestDemoFlow.tsx

```diff
@@ -0,0 +1,123 @@
+import { useEffect } from 'react';
+import styled from '@emotion/styled';
+import { Skeleton, useMantineColorScheme } from '@mantine/core';
+import { useResizeObserver } from '@mantine/hooks';
+import ReactFlow, { Edge, Node, useReactFlow } from 'react-flow-renderer';
+
+import { colors } from '../../design-system';
+import { TriggerNode } from './TriggerNode';
+import { DigestNode } from './DigestNode';
+import { EmailNode } from './EmailNode';
+import { useTemplateFetcher } from '../../api/hooks';
+import { DigestDemoFlowProvider } from './DigestDemoFlowProvider';
+
+export function DigestDemoFlow({
+  isReadOnly = true,
```

Comment:
this component could be static and dynamic where there is a trigger button and digest interval etc.